docs/proposals/scheduler/README.md

```diff
@@ -0,0 +1,144 @@
+
+# Orka - Cluster Scheduler
+
+## Requirements
+
+### Service provided
+
+The Scheduler is responsible for the scheduling of nodes and workloads.
+It is the interface between the Controller and the Agent.
+
+### Scope
+
+The Scheduler is reponsible for:
+
+- Sending workload scheduling/re-scheduling
+    - Re-scheduling can happen upon Agent disconnection
+- Nodes and Workload Instances status updates
+    - received from the Agents
+    - sent to the Controller
+- Handling of Agent connection
+    - the Scheduler is responsible for the handshake
+    - secure connection should be made between Agent and Scheduler
```

Comment:
should or must?

Reply:
[Done](https://github.com/dev-sys-do/orka/pull/2/commits/c05d47b3a14bf86d2dfef03cd4951d34367a2390#diff-a921ddd7f6a17dde09ca710e0254a45c8cf87c34b6872502dc8c80d5ab43c556R21)! It's a "must".